.zenodo.json

```diff
@@ -78,7 +78,102 @@
       "orcid": "0000-0001-8421-365X"
     },
     {
-     "affiliation": "Université Paris-Saclay, Inria, CEA, Palaiseau, France",
+      "affiliation": "Department of Psychology, University of Washington, Seattle WA, USA",
+      "name": "Ariel Rokem",
+      "orcid": "0000-0003-0679-1985"
+     },
+     {
+      "affiliation": "Department of Psychology, University of Washington, Seattle WA, USA",
+      "name": "Daniel McCloy",
+      "orcid": "0000-0002-7572-3241"
```

Comment:
@drammock is this correct?

Reply:
Did you just add me manually? I don't think I've ever contributed. I was just filling in Ariel's info because I happened to know it.

Reply:
If I'm wrong, then it should be
> Institute for Learning & Brain Sciences, University of Washington, Seattle WA, USA

Reply:
I added you manually. You going over the PR and trying to help was enough for me (also, we have asked you for help several times for e.g., the website build). If you **don't** want to be listed, that's fine as well, just let me know.

Reply:
no objection to being included. thanks